src/components/FileUpload/FileUpload.module.scss

```diff
@@ -38,6 +49,7 @@
 }
 
 .sizeText {
+  color: $brand-grey;
   margin: 0 5px;
```

Comment:
@corwinharrell do we need this margin to be a variable?

Reply:
Yeah we can change this.